build/helm/edgemesh/charts/server/templates/04-configmap.yaml

```diff
@@ -20,4 +20,12 @@ data:
       tunnel:
         enable: {{ .Values.modules.tunnel.enable }}
         listenPort: {{ .Values.modules.tunnel.listenPort }}
+        {{- if .Values.publicIP }}
```

Comment:
Configuration items are not equivalent to APIs, so it is not necessary to keep publicip. I suggest deleting it directly

Reply:
I have delete all the `PublicIP code`